prac2-pages/Simranpreet Kaur/login2.dart

```diff
@@ -0,0 +1,166 @@
+
+          SizedBox(height: 20),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(0,0,210,0),
+            child: Container(
+              padding: EdgeInsets.all(20),
+              decoration: BoxDecoration(
+                  color: Color(0xFF06061C),
+                  borderRadius: BorderRadius.circular(15)
+                  ),
+                height : 50,
+                width : 150,
+              child:  SizedBox(
+                width: 600.0,
+                height: 250.0,
+                child: ElevatedButton(
+                    style: ElevatedButton.styleFrom(
+                      primary: Color(0xFF06061C),
+                    ),
+                    child: const Text('Send Code',
+                      style: TextStyle(fontSize: 12,
+                      ),),
+                    onPressed: () {
+
+                    },
+                  ),
+              ),
+
+            ),
+          ),
+          SizedBox(height: 70),
+              Padding(
+                padding: const EdgeInsets.only(left:20.0),
+                child: Row(
+                  children: [
+                    Transform.scale(
+                        scale: 1.3,
+                        child: Checkbox( //only check box
+                            activeColor: Color(0xFF06061C),
+                            value: val2, //unchecked
+                            onChanged: (bool? value){
+                              //value returned when checkbox is clicked
+                              setState(() {
+                                val2 = value!;
+                              });
+                            }
+                        )
+                    ),
+                    Text(
+                        "   Remember this device",
+                        style: TextStyle(fontSize: 15.0,color:Color(0xFF06061C),fontWeight: FontWeight.bold, ),
+
+                    ),],),),
+                    Padding(
+                      padding: const EdgeInsets.fromLTRB(50,0,25,10),
+                      child: Text(
+                        'Don\'t select this option if you are using a public computer.',
+                        textAlign: TextAlign.left,
+                        style: TextStyle(
+
+                          color: Colors.grey[700],
+                          fontSize: 15,
+                        ),
+                      ),
+                    ),
+              SizedBox(height: 80,),
+              TextButton(
+                style: TextButton.styleFrom(
+                  foregroundColor:  Colors.white,
+                ),
+                child: const Text('Return to Login',
+                  style: TextStyle(fontSize: 15,color: Colors.deepPurpleAccent,
+                  ),),
+                onPressed: () {
+                  Navigator. of(context). push(MaterialPageRoute(builder: (context)=>LoginPage()));
+                },
+              ),
+          ],
+        ),
+      ),),);
+
+
+  }
```

Comment:
### Issue:
There is a Bottom Pixel overflow error in this page. 

- This occurs because the sizes that you have specified were specific to your own device screen, but in different devices they were not proper.
- You have to make create a **pixel-perfect and adaptive page** generally, so as to support multiple viewports
- One such package to help you do that is **ScreenUtil**. 

**Screenshot of error**
![WhatsApp Image 2022-12-23 at 12 34 09](https://user-images.githubusercontent.com/106133014/209289201-f7f4c12a-60c8-486e-a532-98f2e3503b5f.jpeg)

Reply:
Resolved! :+1: